components/artist_page_cta/view.coffee

```diff
@@ -53,10 +53,18 @@ module.exports = class ArtistPageCTAView extends Backbone.View
       afterAuthPath: @afterAuthPath
     @$(".artist-page-cta-overlay__register input[name='name']").focus()
     @$('.artist-page-cta-overlay__close').on 'click', @closeOverlay
+    @disableScroll()
+
+  disableScroll: ->
+    $('#main-layout-container').addClass('artist-page-cta__no-scroll')
```

Comment:
I guess I'm surprised that we don't already have a class like that for modals but OK, fair enough 🥇